src/styles/index.scss

```diff
@@ -1,3 +1,273 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  --hover-color: var(--blue);
+
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  padding: 0 40px;
+  display: flex;
+  align-items: center;
+  border: 2px solid var(--blue);
+  justify-content: space-between;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-right: 50px;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link.is-active {
+  color: var(--hover-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  height: 4px;
+  background-color: var(--hover-color);
+  border-radius: 8px;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  position: relative;
+  display: flex;
+  line-height: 60px;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #060b35;
+  align-items: center;
+
+  white-space: nowrap;
+}
+
+.nav__link:hover {
+  color: var(--hover-color);
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  margin: 10px;
+  border: 1px solid white;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin-top: 33px;
+  margin-left: 20px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  padding: 40px 16px 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card:hover {
+  border: 1px solid blue;
+  border-radius: 5px;
+  transform: scale(1.2);
+}
```

Comment:
The `transform: scale(1.2);` on the `.card:hover` may cause layout shifts, which can be visually disruptive. If scaling is required, make sure to handle the layout flow to prevent elements from overlapping or causing unexpected shifts.